frontend/packages/dev-console/src/components/pipelineruns/PipelineRunVisualization.tsx

```diff
@@ -21,7 +21,7 @@ export class PipelineRunVisualization extends React.Component<
   constructor(props) {
     super(props);
     this.state = {
-      pipeline: { apiVersion: '', metadata: {}, kind: 'PipelineRun' },
+      pipeline: { apiVersion: '', metadata: {}, kind: referenceForModel(PipelineResourceModel) },
```

Comment:
You don't want `referenceForModel` here. This should be just the kind as before

Reply:
Done.